src/views/Simulation/utils/pixiUtils.js

```diff
@@ -1,35 +1,48 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
 import * as d3 from 'd3';
-import { AdvancedBloomFilter, GlowFilter } from 'pixi-filters';
+import { AdvancedBloomFilter, GlowFilter, SimpleLightmapFilter } from 'pixi-filters';
 import { AlphaFilter, Application, BitmapText, Container, Graphics, GraphicsContext, Sprite } from 'pixi.js';
 import * as PIXI from 'pixi.js';
 import 'pixi.js/unsafe-eval';
 import { NODE_TYPES } from '../constants/nodeLabels';
 import worldGeoJSON from '../data/map/custom.geo.json';
+import { simulationTheme } from '../theme';
 import { MinimapContainer } from './MinimapContainer';
 import { SceneContainer } from './SceneContainer';
 import { PACKAGE_ICON_LINES, GEAR_ICON_LINES, FACTORY_ICON_LINES } from './shapes';
 
-const GRAY_LINE_COLOR = 0xb9bac0;
-const LINK_COLOR = 0xf5f3f3;
-const HIDDEN_LINK_COLOR = 0x636363;
-const RED_LINE_COLOR = 0xdf3537;
 const COUNTRY_COLOR = 0x333333;
 const BORDER_COLOR = 0x757272;
-const DEFAULT_TEXT_STYLE = { fontFamily: 'Arial', fontSize: 12, fill: 0xffffff, align: 'center' };
-const BLOOM_FILTER = new AdvancedBloomFilter({ blur: 1, quality: 16, bloomScale: 1.8, brightness: 1 });
+const DARK_BLOOM_FILTER = new AdvancedBloomFilter({ blur: 1, quality: 16, bloomScale: 1.8, brightness: 1 });
+const LIGHT_BLOOM_FILTER = new GlowFilter({ distance: 10, outerStrength: 6, color: 0xffffff });
 const SELECTED_LINK_FILTER = new GlowFilter({ distance: 10, outerStrength: 6, color: 0xffffff });
-const SELECTED_NODE_FILTER = new AdvancedBloomFilter({
+const DARK_SELECTED_NODE_FILTER = new AdvancedBloomFilter({
   threshold: 0.1,
   bloomScale: 1.2,
   brightness: 1.5,
   blur: 1,
   quality: 20,
 });
+const LIGHT_SELECTED_NODE_FILTER = new SimpleLightmapFilter({
+  alpha: 0.4,
+  color: 0xffffff,
+});
 const NODE_LABEL_LINE_LENGTH = 24;
 
-const createLabel = (value, forceSplit = true) => {
+export const isDarkMode = (mode) => mode === 'dark';
+
+const getBloomFilter = (mode) => (isDarkMode(mode) ? DARK_BLOOM_FILTER : LIGHT_BLOOM_FILTER);
+const getSelectedNodeFilter = (mode) => (isDarkMode(mode) ? DARK_SELECTED_NODE_FILTER : LIGHT_SELECTED_NODE_FILTER);
+
+const getDefaultStyleText = (palette) => ({
+  fontFamily: 'Arial',
+  fontSize: 12,
+  fill: palette.text,
+  align: 'center',
+});
```

Comment:
Not sure there is a real benefit to move this structure in a function that is used only once 😅 
We can probably just keep it in the `createLabel` function, or move the parts of the style that are constant in a `const` declared at the top of the file (but a function just for this is overkill 🙂)